src/include/migraphx/op/common.hpp

```diff
@@ -15,6 +17,19 @@ enum padding_mode_t
     valid
 };
 
+// The pooling modes must correspond 1-1 to the operators defined for struct parse_pooling.
+enum class pooling_mode
+{
+    avg,
+    max
+};
+
+enum class roialign_mode
```

Comment:
This enum should be removed. The mode in roialign is describing the pooling_mode so it should use the same `pooling_mode` enum. This should also make it clearer in the code that the mode is the pooling mode and not some other type of mode.